packaging/os/pacman.py

```diff
@@ -233,14 +239,19 @@ def install_packages(module, pacman_path, state, packages, package_files):
 
     if install_c > 0:
         module.exit_json(changed=True, msg="installed %s package(s)" % (install_c))
+    
+    if install_c > 0:
+        module.exit_json(changed=True, msg="installed %s package(s)" % (install_c))
 
-    module.exit_json(changed=False, msg="package(s) already installed")
+    if state == 'latest' and len(package_err) > 0:
+        additionalmessage = "Install Latest not possible because Version cought not be fetched for: " + ''.join(package_err)
+    module.exit_json(changed=False, msg="package(s) already installed. %s" % (additionalmessage))
```

Comment:
No need to do explicit listing of packages (it can potentially be large, polluting the console). Instead, it should suffice to have some message like: `"package(s) already installed, but could not ensure 'latest' state for %s package(s) as remote version could not be fetched."`